src/dotnet-serve/SimpleServer.cs

```diff
@@ -44,6 +44,9 @@ class SimpleServer
         [Option("--path-base <PATH>", Description = "The base URL path of postpended to the site url.")]
         public string PathBase { get; private set; }
 
+        [Option("--default-extensions:<EXTENSIONS>", CommandOptionType.SingleOrNoValue, Description = "A comma-delimited list of extensions to use when no extension is provided in the URL. [.html,.htm]")]
```

Comment:
I'm sure about this syntax. Comma-separated lists can be a pain in powershell - requires escaping. I was thinking instead that we should do `CommandOptionType.MultipleValue`. I wouldn't expect people to have more than one or two default extensions, but for those who do, I planning on adding support for storing dotnet-serve config in a file. Would this be okay with you?

Reply:
I’m not attached to a particular syntax one way or the other. Would that allow the argument to use a default set of values when none are explicitly specified? Given that the default extensions will likely be `htm` and `html` in almost every case I think it makes sense to default to those. I like the idea of being able to specify additional/alternate ones, but not if it means having to specify the most common ones every time.

Reply:
I thought about this more too. Let's go with this for now. If ppl have to escape in powershell, so be it.